gtdynamics/universal_robot/Joint.h

```diff
@@ -75,6 +75,10 @@ struct JointParams {
 
 /// Joint is the base class for a joint connecting two Link objects.
 class Joint : public boost::enable_shared_from_this<Joint> {
+
+  /// Robot class should have access to the internals of its joints.
```

Comment:
Why?

Reply:
Logically makes sense right? A robot should be able to inspect its joints and links.